src/databricks/sql/thrift_api/TCLIService/ttypes.py

```diff
@@ -2841,6 +2907,270 @@ def __ne__(self, other):
         return not (self == other)
 
 
+class TDBSqlJsonArrayFormat(object):
+    """
+    Attributes:
+     - compressionCodec
+
+    """
+
+
+    def __init__(self, compressionCodec=None,):
+        self.compressionCodec = compressionCodec
+
+    def read(self, iprot):
+        if iprot._fast_decode is not None and isinstance(iprot.trans, TTransport.CReadableTransport) and self.thrift_spec is not None:
```

Comment:
Is there any chance that we should not be exposing these Thrift commands in this public repository? Is this new command used anywhere in the connector?

Reply:
I don't think this one is, but this is kind of on the people who edited the original thrift right? I assume this is a feature that is hypothetically useful to a thrift server at some point.

Reply:
Also, what is the reason for keeping the version of our thrift compiler? I've downloaded the latest stable release; I'm not sure how to even download an older version.